test/e2e/Dockerfile

```diff
@@ -56,8 +65,6 @@ ARG TINI_VERSION=0.18.0
 ARG DOCTL_VERSION=1.76.2
 # ginkgo is needed to run the upstream end-to-end tests.
 ARG GINKGO_VERSION=1.10.3
-# kubectl is needed by some tests.
-ARG KUBECTL_VERSION=1.23.7
```

Comment:
Is this not needed anymore?

Reply:
I removed it, because it was not used. In l. 78ff. we just install the latest stable version.

Reply:
👍